workbench-core/datasets/src/models/addDataSetExternalEndpoint.ts

```diff
@@ -3,19 +3,21 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { AuthenticatedUser } from '@aws/workbench-core-authorization';
+import { AuthenticatedUser, IdentityType } from '@aws/workbench-core-authorization';
```

Comment:
Do not import `authenticatedUser` from authorization in the `models/` as it tightly couples implementations. Better to redefine our expected `authenticatedUser` so we can eventually refactor to the point that `DataSets` can be consumed without the `@aws/workbench-core-authorization` dependency.

Reply:
I am fine with `DataSets` defining its own `authenticatedUser` object. It would also be okay to pull it from `base`, but it may be a bit difficult to discover there.

Reply:
Wont that put us at risk of things breaking if the implementation of either of these changes? Could a solution be to re-export external package public facing interfaces in the index file? So no other package is needed